tests/test_assignment_type.py

```diff
@@ -45,14 +37,9 @@ def test_stochastic():
     OUTPUT_DIR     = os.path.join(EXAMPLES_DIR,"output")
 
     scenario_dir = os.path.join(INPUT_NETWORKS, 'simple')
```

Comment:
Comment for throughout the tests:
- since a scenario is defined by networks, demand, and parameters...would prefer that `input_network_dir`  refer directly to the folder that contains the GTFS-PLUS files.  This would be consistent with `input_demand` referring to the directory the demand files are in rather than a shell directory.

Why? I want to reserve the word scenario :-)